freshbooks/api/business.py

```diff
@@ -0,0 +1,181 @@
+from decimal import Decimal
+from types import SimpleNamespace
+from typing import Any, List, Optional, Tuple, Union
+
+from freshbooks.api.resource import HttpVerbs, Resource
+from freshbooks.builders import Builder
+from freshbooks.builders.includes import IncludesBuilder
+from freshbooks.errors import FreshBooksError, FreshBooksNotImplementedError
+from freshbooks.models import ListResult, Result, VisState
+
+
+class BusinessResource(Resource):
```

Comment:
I think maybe `AccountingBusinessResource` to make the use clear.

Reply:
This item has been resolved, made appropriate changes